src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts

```diff
@@ -1745,6 +1764,8 @@ class ExtensionStorageCleaner implements IWorkbenchContribution {
 	}
 }
 
+registerWorkbenchContribution2(ExtensionsSettingsContributions.ID, ExtensionsSettingsContributions, WorkbenchPhase.BlockStartup);
```

Comment:
If this is just registering configs could this be deferred and not block startup?

Reply:
They were registered globally before - hence no change from before. Also some of these are needed earlier.